src/component/child/index.js

```diff
@@ -76,6 +80,23 @@ export class ChildComponent extends BaseComponent {
         });
     }
 
+    hasValidParentDomain() {
+        const parentDomain = this.getParentDomain();
+        if (typeof this.component.allowedParentDomains === 'string') {
```

Comment:
Thanks for the matchDomain tests! Added and fixed in cross-domain-utils, can you try with the latest version?